### ISSUE TITLE
Can we import instrument-specific XML attributes file?

### ISSUE THREAD

**Issue by prjemian** [areaDetector/ADCore#511]
We want a main attributes file that imports from a detector-specific file.  Is this possible now?  What work would it take to add it?

**Comment by EmilioPeJu** [comment]
Do you mean if including another file is supported?

**Comment by prjemian** [comment]
Yes, where the main attributes file imports another attributes file
specific to that detector.

**Comment by MarkRivers** [comment]
@prjemian what is the syntax for that? Are there any examples in AdSimDetector or elsewhere?

**Comment by mp49** [comment]
We handle this by using an external script that builds multiple XML files, each one for a different detector. 

Although, there seems to be built-in XML support for this:
https://stackoverflow.com/questions/5121052/can-we-import-xml-file-into-another-xml-file
